meshio/vtu/_vtu.py

```diff
@@ -689,6 +830,16 @@ def text_writer(f):
                 key_ = k[:7]
             elif k[:10] == "polyhedron":
                 key_ = k[:10]
+                # Get face-cell relation on the vtu format. See comments in helper
+                # function for more information of how to specify this.
+                faces_loc, faceoffsets_loc = _polyhedron_face_cells(
+                    mesh.polyhedron_faces[k]
+                )
+                # Adjust offsets to global numbering
+                if faceoffsets.size > 0:
+                    faceoffsets_loc += faceoffsets[-1]
+                faces = numpy.append(faces, faces_loc)
```

Comment:
Better collect all faces in a list, and at the end call a `numpy.concatenate`.

Reply:
Implemented.